JavaScript/JS-Week_1/Javascript_Homework/Ez Namey/script.js

```diff
@@ -0,0 +1,17 @@
+
+const firstWords = ["Easy", "Awesome", "Corporate", "Innovative", "Dynamic", "Creative", "Smart", "Tech", "NextGen", "Future"];
+const secondWords = ["Solutions", "Corporation", "Enterprises", "Technologies", "Systems", "Developments", "Networks", "Ventures", "Innovations", "Dynamics"];
+
+const randomNumber = Math.floor(Math.random() * 10);
```

Comment:
Using only one random number would result in always getting the same pairs of names. To bring more variety, you could define `randomNumber2` which you could use for with `secondWords[randomNumber2]`.

Reply:
Thank you for your suggestion . I have added new variable to get better name.